lib/ingram_micro/transmission.rb

```diff
@@ -28,6 +28,20 @@ def schema_valid?
     valid
   end
 
+  # This will find the appropriate xsd file and return it for use in xml
+  # validation. For sales-order-submission, there are two different schemas,
+  # and which one to load depends on whether you need the one configured for
+  # domestic-only use or the one for mixed domestic and internatinal use.
+  def load_schema
```

Comment:
i dont really understand this method, why the roundabout way of setting transmission_name? can't you jsut say:

`transmission_name = IngramMicro.domestic_schema? ? self.class::TRANSMISSION_FILENAME : 'sales-order-international'`
OR
`transmission_name = IngramMicro.domestic_schema? ? 'sales-order-domestic' : 'sales-order-international'`